### ISSUE TITLE
BioEco / OCG / 

### ISSUE THREAD

**Issue by pbuttigieg** [iodepo/odis-arch#402]
...

Magali Krieger (metadat expert)  mkrieger@ocean-ops.org
Thomas Latter tlatter@ocean-ops.org (front end and API developer)

**Comment by pbuttigieg** [comment]
![image](https://github.com/iodepo/odis-arch/assets/2276146/974bfbeb-7925-42fa-a369-1e4756336418)

**Comment by pbuttigieg** [comment]
A little rewiring in a follow-up with @pieterprovoost

The main change is to better show that OBIS and its BioEco Portal will/already do serve similar functions to OceanOps services, but focused on biodiversity (meta)data flows. 

Through ODIS interoperation, all these modules can embed and leverage one another. 

![image](https://github.com/iodepo/odis-arch/assets/2276146/a008a9ca-5808-4fa7-9595-a8b511a7b984)

**Comment by wardappeltans** [comment]
The 10-odd GOOS networks are connected to OceanOPS, but in the BioEco world we do not have GOOS networks, but a zillion BioEco obs programmes and projects. So perhaps there should be a parallel flow from those programmes via the OBIS nodes network including central OBIS office. The OBIS data system can be placed at the same level as the ERDDAP.

**Comment by pbuttigieg** [comment]
> The 10-odd GOOS networks are connected to OceanOPS, but in the BioEco world we do not have GOOS networks, but a zillion BioEco obs programmes and projects. So perhaps there should be a parallel flow from those programmes via the OBIS nodes network including central OBIS office. The OBIS data system can be placed at the same level as the ERDDAP.

From the ODIS point of view, that would make sense and also simplify the main data flow - if OBIS could be the hub, it would simplify things greatly. Of course, any system could connect to ODIS, share the EOV patterns, and be detectable by the BioEco Portal too, if they wanted / needed to.

**Comment by jmckenna** [comment]
I'm curious to hear from the technical teams how they each wish to pull from ODIS, I guess that can be discussed at the next technical meeting (direct SPARQL queries to the graph have proven to be so heavy, for example).
PS> I think we should start using these workflow/Vue diagrams more often, very insightful.

**Comment by pbuttigieg** [comment]
> I'm curious to hear from the technical teams how they each wish to pull from ODIS, I guess that can be discussed at the next technical meeting (direct SPARQL queries to the graph have proven to be so heavy, for example).
> PS> I think we should start using these workflow/Vue diagrams more often, very insightful.

Yes, my assumption is that they'll go for parquet or right at the JSON-LD / objects. We must then document more clearly how this can be done @fils
